BlockServer/core/inactive_config_holder.py

```diff
@@ -73,8 +73,11 @@ def set_config_details(self, details):
                     if ioc.get('component') is not None:
                         raise ValueError('Cannot override iocs from components')
 
+                    remotePvPrefix = ioc.get("remotePvPrefix")
```

Comment:
constant please

Reply:
Doing this makes the file very messy, i.e. the whole file would start with a large block of
```
BLOCKS="blocks"
IOCS="iocs"
REMOTE_PV_PREFIX="remotePvPrefix"
GROUPS="groups"
...
```
I'm not sure this makes the code any nicer. If you disagree let me know and I can change it...